.github/actions/setup/action.yaml

```diff
@@ -8,7 +8,7 @@ inputs:
   helm-version:
     description: "The helm version to be used when setting up"
     required: false
-    default: "v3.9.4"
+    default: "v3.14.4"
```

Comment:
I'd prefer not to update the default for all charts since the others can still use 3.9 (and I know people are still using it).  Can you since update the test demo yaml to set this value?

Reply:
I changed this to set the version in demo-test.yaml instead, and reverted to 3.9.4 in the shared setup action.